online/main.yaml

```diff
@@ -148,14 +148,30 @@ sources:
-      license: CC-BY-4.0
+      title: IFS-FESOM 2.8km simulation (DEEPOFF). Access via EERIE cloud.
+      summary: |
+        Coupled simulation at 2.8 km resolution, with deep convection parameterization switched off. Run for 1 year and 2 months, starting in 2020.
+
+        **General**:
+          * The Integrated Forecasting System (IFS), developed at ECMWF, is a spectral-transform atmospheric model with two-time-level semi-implicit, semi-Lagrangian time stepping (Temperton et al., 2001; Hortal, 2002; Diamantakis and Váňa, 2021). It is coupled to other Earth system components (land, waves, ocean, sea ice).
+          * The global hackathon simulations at 2.8km are with the IFS-FESOM configuration (Rackow et al. 2025), coupled to the FESOM2 ocean-sea ice model.
+
+        **Simulation**:
+          * This is one of two new IFS simulations at 2.8 km resolution, coupled to FESOM 5 km. The simulations are started on 01.01.2020 and run for 14 months, ending on 01.03.2021. The two simulations differ in the treatment of deep convection. This simulation (DEEPOFF) only has the shallow convection scheme active and the deep convection scheme switched off, while the second simulation (RCBMF) has the shallow convection scheme fully active and the deep convection scheme weakly active (with a strongly reduced cloud base mass flux), in agreement with 9 km nextGEMS production simulations.
+
+        **Processing**:
+          * IFS simulations provided via EERIE cloud use IFS naming of variables. Please refer to the [parameter database](https://codes.ecmwf.int/grib/param-db/) for more information on the available variables.
+          * Output is written to HEALPix H2048 (zoom level 11, about 3 km resolution), and to H128 (level 7, about 50 km resolution). The remapping to zoom level 7 used conservative interpolation for the 2D variables and linear interpolation for the 3D variables. Remapping to the high-resolution H2048 (level 11) was also done with linear interpolation. Output is available hourly (PT1H). In addition, we also provide monthly means (P1M), but only for those variables that are accumulated in time (all fluxes, e.g. radiation, precipitation, etc.).
+      license: Creative Commons CC BY 4.0. https://creativecommons.org/licenses/by/4.0/
+      creator_name: Tobias Becker, Thomas Rackow, Xabier Pedruzo-Bagazgoitia
+      creator_email: tobias.becker@ecmwf.int, thomas.rackow@ecmwf.int, xabier.pedruzo@ecmwf.int
       source_id: IFS-FESOM
       simulation_id: DEEPOFF
       time_start: 2020-01-01T00:00:00
       time_end: 2021-03-01T00:00:00
+      references: IFS-FESOM - Rackow, Pedruzo-Bagazgoitia, Becker, Milinski, Sandu et al. (2025) https://doi.org/10.5194/gmd-18-33-2025, IFS model - https://www.ecmwf.int/en/forecasts/documentation-and-support/changes-ecmwf-model, FESOM model - https://fesom.de
```

Comment:
References should *only* contain the links, authors etc...

Reply:
You mean

`references: https://doi.org/10.5194/gmd-18-33-2025, https://www.ecmwf.int/en/forecasts/documentation-and-support/changes-ecmwf-model, https://fesom.de` ?

You have a link for allowed content?

Reply:
Yes.
You can also choose between having a `,` separated string, or an actual yaml-list of strings.

Reply:
OK, done that!